com.hazelcast.jet.MulticastDiscoveryTest.when_jetClientCreated_then_connectsToJetCluster

_3.1-maintenance_ (commit b79453edce91c6ab3bfbf57d7574f89fdb572f01)

Failed on Oracle JDK11: http://jenkins.hazelcast.com/job/jet-oss-maintenance-oracle-jdk11/63/testReport/junit/com.hazelcast.jet/MulticastDiscoveryTest/when_jetClientCreated_then_connectsToJetCluster/

Stacktrace:
```
java.lang.AssertionError: expected:<2> but was:<3>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.jet.MulticastDiscoveryTest.when_jetClientCreated_then_connectsToJetCluster(MulticastDiscoveryTest.java:86)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Standard output:
```
Hiccups measured while running test 'when_jetClientCreated_then_connectsToJetCluster(com.hazelcast.jet.MulticastDiscoveryTest):'
16:24:30, accumulated pauses: 75 ms, max pause: 10 ms, pauses over 1000 ms: 0
16:24:35, accumulated pauses: 287 ms, max pause: 167 ms, pauses over 1000 ms: 0
16:24:40, accumulated pauses: 6 ms, max pause: 1 ms, pauses over 1000 ms: 0
```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/config/XmlJetConfigWithSystemPropertyTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/config/XmlJetConfigWithSystemPropertyTest.java
@@ -17,8 +17,7 @@
 package com.hazelcast.jet.config;
 
 import com.hazelcast.core.HazelcastException;
-import com.hazelcast.jet.Jet;
-import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.impl.config.ConfigProvider;
 import com.hazelcast.jet.impl.config.XmlJetConfigBuilder;
 import com.hazelcast.jet.impl.util.Util;
 import com.hazelcast.test.HazelcastSerialClassRunner;
@@ -265,9 +264,7 @@ public class XmlJetConfigWithSystemPropertyTest extends AbstractJetMemberConfigW
     public void loadingThroughSystemPropertyViaLocator_nonXmlSuffix() {
         System.setProperty(HAZELCAST_JET_CONFIG_PROPERTY, "classpath:hazelcast-jet-foo.bar");
 
-        JetInstance instance = Jet.newJetInstance();
-        JetConfig config = instance.getConfig();
-        instance.shutdown();
+        JetConfig config = ConfigProvider.locateAndGetJetConfig();
 
         assertEquals("bar", config.getProperties().getProperty("foo"));
 

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/config/YamlJetConfigWithSystemPropertyTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/config/YamlJetConfigWithSystemPropertyTest.java
@@ -17,8 +17,7 @@
 package com.hazelcast.jet.config;
 
 import com.hazelcast.core.HazelcastException;
-import com.hazelcast.jet.Jet;
-import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.impl.config.ConfigProvider;
 import com.hazelcast.jet.impl.config.YamlJetConfigBuilder;
 import com.hazelcast.jet.impl.util.Util;
 import com.hazelcast.test.HazelcastSerialClassRunner;
@@ -275,11 +274,7 @@ public class YamlJetConfigWithSystemPropertyTest extends AbstractJetMemberConfig
     public void loadingThroughSystemPropertyWithLocator_existingClasspathResource() {
         System.setProperty(HAZELCAST_JET_CONFIG_PROPERTY, "classpath:" + JET_TEST_YAML);
 
-        JetInstance instance = Jet.newJetInstance();
-        JetConfig config = instance.getConfig();
-        instance.shutdown();
-
-        assertConfig(config);
+        assertConfig(ConfigProvider.locateAndGetJetConfig());
     }
 
 }

